plansys2_executor/src/plansys2_executor/ExecutorClient.cpp

```diff
@@ -80,10 +81,53 @@ ExecutorClient::execute_and_check_plan()
 
   switch (result_.code) {
     case rclcpp_action::ResultCode::SUCCEEDED:
-      if (result_.result != nullptr && result_.result->success) {
+      if (result_.result == nullptr) {
+        RCLCPP_WARN(
+          node_->get_logger(), "Plan failed due to a nullptr in the result");
+      } else if (result_.result->success) {
         RCLCPP_INFO(node_->get_logger(), "Plan Succeeded");
       } else {
-        RCLCPP_INFO(node_->get_logger(), "Plan Failed");
+        RCLCPP_ERROR(node_->get_logger(), "Plan Failed");
+        for (auto msg : result_.result->action_execution_status) {
+          switch (msg.status) {
+            case plansys2_msgs::msg::ActionExecutionInfo::SUCCEEDED:
+              RCLCPP_WARN_STREAM(
```

Comment:
Why WARN and not INFO or DEBUG?

Reply:
that is an error, I will change it.  Thoughts on INFO vs DEBUG?